### PR TITLE
Audit Item L03

### LINKED ISSUE #24: Resolve audit item L03

### PR DESCRIPTION
**Pull request by rheeger**
closes #24

### CODE REVIEW

File: contracts/Org.sol [hunk 1]
```diff
@@ -96,17 +96,29 @@ contract Org is Administratable {
...
+     * @param providedWallet Wallet address that will serve as destination for withdrawl
+     * @param adminContractAddress Contract Address for Endaoment Admin
+     */
     function setOrgWallet(address providedWallet, address adminContractAddress) public onlyAdminOrRole(adminContractAddress, IEndaomentAdmin.Role.REVIEWER){
         orgWallet = providedWallet;
     }
 
+    /**
+     * @notice Retrieves Token Balance of Org Contract
+     * @param tokenAddress Address of desired token to query for balance
```
Comment (CantelopePeel) [inline contracts/Org.sol:99]: Add @returns

File: contracts/Org.sol [hunk 2]
```diff
@@ -96,17 +96,29 @@ contract Org is Administratable {
...
+     */
      function getTokenBalance(address tokenAddress) public view returns (uint) {
             ERC20 t = ERC20(tokenAddress);
             uint256 bal = t.balanceOf(address(this));
 
         return bal;
      }
 
+    /**
+     * @notice Retrieves Count of Claims Made
```
Comment (CantelopePeel) [inline contracts/Org.sol:110]: Ditto

File: contracts/Org.sol [hunk 3]
```diff
@@ -96,17 +96,29 @@ contract Org is Administratable {
         emit cashOutComplete(cashOutAmount);
     }
 
+    /**
+     * @notice Assigns orgWallet to Provided Address
+     * @param providedWallet Wallet address that will serve as destination for withdrawl
```
Comment (CantelopePeel) [inline contracts/Org.sol]: s/withdrawl/withdrawal

**Review by CantelopePeel — CHANGES_REQUESTED** [review]

File: contracts/Org.sol (hunk 3 above)
Comment (rheeger) [reply contracts/Org.sol]: fixed

File: contracts/Org.sol (hunk 2 above)
Comment (rheeger) [reply contracts/Org.sol:110]: fixed

File: contracts/Org.sol (hunk 1 above)
Comment (rheeger) [reply contracts/Org.sol:99]: fixed

**Review by CantelopePeel — APPROVED** [review]
LGTM

**Pull request merged**